src/System.Collections.Immutable/src/System.Collections.Immutable.csproj

```diff
@@ -9,16 +9,14 @@
     <FileAlignment>512</FileAlignment>
     <DocumentationFile>$(OutputPath)$(MSBuildProjectName).xml</DocumentationFile>
     <GenerateAppxPackageOnBuild>False</GenerateAppxPackageOnBuild>
-    <PackageTargetFramework Condition="'$(TargetGroup)' == 'netstandard1.0'">netstandard1.0;portable-net45+win8+wp8+wpa81</PackageTargetFramework>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Release|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Release|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard1.0-Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard1.0-Release|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap-Windows_NT-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(TargetGroup)' == 'netstandard1.0'">
```

Comment:
I don't think we need these. These are added by our infrastructure when we create the package. When we build allConfigurations, since you added netstandard1.3 to the `<PackageConfigurations>` it will target the package for netstandard1.3 as well and will add the reference assembly to the ref path for that framework. So no need of adding the PackageTargetFramework.

As you can see we also already define them in the pkgproj: https://github.com/dotnet/corefx/blob/master/src/System.Collections.Immutable/pkg/System.Collections.Immutable.pkgproj#L10

Reply:
I'll remove it in that case.

---
In reply to: [157077750](https://github.com/dotnet/corefx/pull/25738#discussion_r157077750) [](ancestors = 157077750)